gtsam/nonlinear/ISAM2Result.h

```diff
@@ -175,6 +175,7 @@ struct ISAM2Result {
   /** Getters and Setters */
   size_t getVariablesRelinearized() const { return variablesRelinearized; }
   size_t getVariablesReeliminated() const { return variablesReeliminated; }
+  FactorIndices getNewFactorsIndices() const { return newFactorsIndices; }
```

Comment:
Will MATLAB work with this?

Reply:
It should because FactorIndices is also a wrapped class 😄